frontend/appflowy_flutter/lib/mobile/presentation/database/field/mobile_edit_field_screen.dart

```diff
@@ -49,76 +49,70 @@ class _MobileEditPropertyScreenState extends State<MobileEditPropertyScreen> {
     final viewId = widget.viewId;
     final fieldId = widget.field.id;
 
-    return Scaffold(
-      appBar: AppBar(
-        centerTitle: true,
-        title: FlowyText.medium(
-          LocaleKeys.grid_field_editProperty.tr(),
-        ),
-        elevation: 0,
-        bottom: const PreferredSize(
-          preferredSize: Size.fromHeight(1),
-          child: Divider(
-            height: 1,
-            thickness: 1,
+    return PopScope(
+      onPopInvoked: (didPop) {
+        if (didPop) {
+          context.pop(_fieldOptionValues);
+        }
+      },
+      child: Scaffold(
+        appBar: AppBar(
+          centerTitle: true,
+          title: FlowyText.medium(
+            LocaleKeys.grid_field_editProperty.tr(),
+          ),
+          elevation: 0,
+          bottom: const PreferredSize(
+            preferredSize: Size.fromHeight(1),
+            child: Divider(
+              height: 1,
+              thickness: 1,
+            ),
+          ),
+          leading: AppBarBackButton(
+            onTap: () => context.pop(_fieldOptionValues),
           ),
         ),
-        leading: AppBarBackButton(
-          onTap: () => context.pop(field),
-        ),
-      ),
-      body: FieldOptionEditor(
-        mode: FieldOptionMode.edit,
-        isPrimary: widget.field.isPrimary,
-        defaultValues: field,
-        actions: [
-          if (widget.field.fieldSettings?.visibility.isVisibleState() ?? true)
-            FieldOptionAction.hide
-          else
-            FieldOptionAction.show,
-          FieldOptionAction.duplicate,
-          FieldOptionAction.delete,
-        ],
-        onOptionValuesChanged: (newField) async {
-          if (newField.name != field.name) {
-            await fieldService.updateField(name: newField.name);
-          }
-
-          if (newField.type != widget.field.fieldType) {
-            await fieldService.updateType(fieldType: newField.type);
-          }
-
-          final data = newField.getTypeOptionData();
-          if (data != null) {
-            await FieldBackendService.updateFieldTypeOption(
+        body: MobileFieldEditor(
+          mode: FieldOptionMode.edit,
+          isPrimary: widget.field.isPrimary,
+          defaultValues: _fieldOptionValues,
+          actions: [
+            if (widget.field.fieldSettings?.visibility.isVisibleState() ?? true)
+              FieldOptionAction.hide
+            else
+              FieldOptionAction.show,
```

Comment:
I know it's not included in this PR, but I suggest using the operator `? :` instead of `if-else` here for better readability.